examples/instances_fixed_shape/variables.tf

```diff
@@ -131,6 +131,12 @@ variable "subnet_ocids" {
 
 # storage parameters
 
+variable "boot_volume_backup_policy" {
+  description = "Choose between default backup policies : Gold, Silver, Bronze. Use Disabled to affect no backup policy on the Boot Volume."
+  type        = string
+  default     = "disabled"
```

Comment:
Make the case for "disabled the same". I suggest set the possible values to gold, silver, bronze and disabled. You can then handle it in your code. This won't confuse the user then.

Reply:
Also add validation?

Reply:
Case corrected: all lowercase everywhere.

Validation is already handled in the module, like this:
```hcl
variable "boot_volume_backup_policy" {
  description = "Choose between default backup policies : gold, silver, bronze. Use disabled to affect no backup policy on the Boot Volume."
  type        = string
  default     = "disabled"

  validation {
    condition     = contains(["gold", "silver", "bronze", "disabled"], var.boot_volume_backup_policy)
    error_message = "Accepted values are gold, silver, bronze or disabled (case sensitive)."
  }
}
```

I think there is no really point to set it again in the customer config, as these are only examples and the code could vary a lot based on the usage.

Note: In a next iteration of this feature, validation condition will need to evolve to authorize custom backup policy ids.